core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -113,7 +142,7 @@ export class TranslationModalComponent {
     this.HTML_SCHEMA = {
       type: 'html',
       ui_config: {
-        hide_complex_extensions: true,
+        hide_complex_extensions: false,
```

Comment:
Had to make this false as some of the extensions which were enabled in the exploration editor is disabled by making this true. Hence, the validation to check all the elements which are present in the original text are added to the translation, is not possible as the translator can not add those hidden elements in the translation.

Reply:
According to what it is being said [here](issue-translation-copy-new), I do not think it is necessary to hide complex elements in translation editor particularly as they are not hidden in the exploration editor. If we hide these elements in the translation editor, then we won't be able to validate translations which has original texts with those complex extensions as the translator can not include those complex extensions if we hide them.

Reply:
Okay I think I understand. But definitely not the expert here!